crates/sui-rosetta/src/construction.rs

```diff
@@ -271,8 +271,9 @@ pub async fn metadata(
         })?;
     let dry_run = context.client.read_api().dry_run_transaction(data).await?;
 
-    let budget =
-        dry_run.effects.gas_used().computation_cost + dry_run.effects.gas_used().storage_cost;
+    let budget = dry_run.effects.gas_used().computation_cost
+        + dry_run.effects.gas_used().storage_cost
+        + 1000;
```

Comment:
added this hack to get rosetta cli to pass. Every tet (rosetta and not) passes and unit test in rosetta are good too so I am a bit at a loss as to why this is happening.
I'd like to hack it this way for now and get back to this